_data/tools.yml

```diff
@@ -276,6 +276,15 @@
   v3: true
 
 # Mock Servers
+- name: Connexion
+  category: mock
+  language: Python
+  link: https://connexion.readthedocs.io/en/latest/
+  github: https://github.com/zalando/connexion
+  description: OpenAPI First framework for Python on top of Flask with automatic endpoint validation & OAuth2 support
+  v2: true
+  v3_progress_link: true
```

Comment:
Your docukmentation says it supports v3.0, does it not? 

If it supports v3, please change this to `v3: true`, if it does not, please change this to `v3_progress_link: http://github.com/issues...` which should be some sort of page explaining the progress towards v3 support. 

Also I just made this PR for you, I'd appreciate it if you could merge :D https://github.com/zalando/connexion/pull/813